lib/install/web.rb

```diff
@@ -0,0 +1,13 @@
+css = ask("What CSS framwork do you want to use? [postcss, tailwind]")
+
+raise ArgumentError if ["postcss","tailwind"].exclude? css
+
+after_bundle do
+  gem_group :development, :test do
+    gem "suspenders", github: "thoughtbot/suspenders", branch: "suspenders-3-0-0-web-generator"
+  end
+
+  run "bundle install"
+
+  generate "suspenders:install:web --css=#{css}"
```

Comment:
We might want to then `cd` into the directory and call `bin/setup` once that generator is created.